### PR TITLE
Feat/v2 tabs

### PR DESCRIPTION
**Pull request by blumendorf**

### CODE REVIEW

File: src/components/V2Tabs/V2Tab.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+describe('Notifications Render Correctly', () => {
+  test('renders active Tab', () => {
+    const { getByText } = render(
+      <V2Tab active label="tab label" value="tab1" />,
+    );
+    getByText('tab label');
+    const tab = getByText('tab label');
+    expect(tab.classList.contains(oltStyles.V2TabsLink)).toBeTruthy();
+    expect(tab.classList.contains(oltStyles.isActive)).toBeTruthy();
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tab.test.js]:
```suggestion
    expect(tab.classList.contains(oltStyles.isActive)).toBe(true);
```
https://vincenttunru.com/toBeTruthy-vs-toBe-true/

File: src/components/V2Tabs/V2Tab.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  });
+  test('renders inactive Tab', () => {
+    const { getByText } = render(
+      <V2Tab active={false} label="tab label" value="tab1" />,
+    );
+    getByText('tab label');
+    const tab = getByText('tab label');
+    expect(tab.classList.contains(oltStyles.V2TabsLink)).toBeTruthy();
+    expect(!tab.classList.contains(oltStyles.isActive)).toBeTruthy();
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tab.test.js]:
```suggestion
    expect(tab.classList.contains(oltStyles.isActive)).toBe(false);
```

File: src/components/V2Tabs/V2Tab.test.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      <V2Tab
+        active={false}
+        label="tab label"
+        value="tab1"
+        onSelect={onSelect}
+      />,
+    );
+    const tab = getByText('tab label');
+    fireEvent.click(tab);
+    expect(onSelect).toHaveBeenCalled();
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tab.test.js]:
```suggestion
    expect(onSelect).toHaveBeenCalledTimes(1);
```

File: src/components/V2Tabs/V2Tabs.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    <div {...rest} style={{ position: 'relative' }} onScroll={handleScroll}>
+      <div
+        className={classnames(
+          !gradient.right && olt.V2Tabs,
+          gradient.right && olt.V2TabsForceScroll,
+          gradient.left && olt.hasGradientLeft,
+          gradient.right && olt.hasGradientRight,
+        )}
+        ref={tabBar}
+        data-testid="Tabs"
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.js]:
```suggestion
```

File: src/components/V2Tabs/V2Tabs.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+
+Simple Tabs example
+
+```js
+import { TaV2bs, V2Tab } from '@lightelligence/react';
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.md]:
```suggestion
import { V2Tabs, V2Tab } from '@lightelligence/react';
```

File: src/components/V2Tabs/V2Tabs.test.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    const { getByText } = renderTabs(() => {});
+    getByText('Tab A');
+    getByText('Tab B');
+    getByText('Tab C');
+    getByText('Tab D');
+    getByText('Tab E');
+    getByText('Tab F');
+    const tabA = getByText('Tab A');
+    expect(tabA.classList.contains(oltStyles.V2TabsLink)).toBeTruthy();
+    expect(tabA.classList.contains(oltStyles.isActive)).toBeTruthy();
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.test.js]:
```suggestion
    expect(tabA.classList.contains(oltStyles.isActive)).toBe(true);
```

File: src/components/V2Tabs/V2Tabs.test.js [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    const tabB = getByText('Tab B');
+    expect(tabB.classList.contains(oltStyles.V2TabsLink)).toBeTruthy();
+    expect(!tabB.classList.contains(oltStyles.isActive)).toBeTruthy();
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.test.js]:
```suggestion
    expect(tabB.classList.contains(oltStyles.isActive)).toBe(false);
```
etc.

File: src/components/V2Tabs/V2Tabs.test.js [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+describe('Tabs render correctly', () => {
+  test('renders Tabs with active Tab', () => {
+    const { getByText } = renderTabs(() => {});
+    getByText('Tab A');
+    getByText('Tab B');
+    getByText('Tab C');
+    getByText('Tab D');
+    getByText('Tab E');
+    getByText('Tab F');
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.test.js]: What's the purpose of these calls? Shouldn't there be an `expect`?

File: src/components/V2Tabs/V2Tabs.test.js [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+    expect(tabA.classList.contains(oltStyles.V2TabsLink)).toBeTruthy();
+    expect(tabA.classList.contains(oltStyles.isActive)).toBeTruthy();
+    const tabB = getByText('Tab B');
+    expect(tabB.classList.contains(oltStyles.V2TabsLink)).toBeTruthy();
+    expect(!tabB.classList.contains(oltStyles.isActive)).toBeTruthy();
+  });
+  test('renders inactive Tab', () => {
+    const onSelect = jest.fn();
+    const { getByText } = renderTabs(onSelect);
+    getByText('Tab A');
```
Comment (bra1n) [inline src/components/V2Tabs/V2Tabs.test.js]: No `expect` inside this test.

File: src/components/V2Tabs/V2Tabs.js (hunk 4 above)
Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.js]: I am actually using that in the test. I can get the element through some dom operations, but this seems more readable.

File: src/components/V2Tabs/V2Tabs.test.js (hunk 8 above)
Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.test.js]: getBy* throws an error if no elements match or if more than one match is found
https://testing-library.com/docs/dom-testing-library/api-queries

File: src/components/V2Tabs/V2Tabs.test.js (hunk 9 above)
Comment (blumendorf) [reply src/components/V2Tabs/V2Tabs.test.js]: getBy* throws an error if no elements match or if more than one match is found

Comment (frabbit) [reply src/components/V2Tabs/V2Tabs.test.js]: yes, getBy is already an expectation

File: src/components/V2Tabs/V2Tabs.test.js (hunk 8 above)
Comment (bra1n) [reply src/components/V2Tabs/V2Tabs.test.js]: Got it, thanks.

**Comment by bra1n** [comment]
handlebars needs to be updated

**Pull request merged**